### ISSUE TITLE
ParserException when new lines occur in yaml

### ISSUE THREAD

**Issue by jhadvig** [jeremyfa/yaml.js#60]
When parsing yaml with new line characters, eg: 
https://github.com/openshift-qe/v3-testfiles/blob/master/image-streams/image-source.yaml#L80
https://github.com/openshift-qe/v3-testfiles/blob/master/image-streams/image-source.yaml#L124-L125
I get back ParseException with message "You cannot define a sequence item when in a mapping", even if its a valid yaml format due to http://www.yamllint.com/

**Comment by jeremyfa** [comment]
Good catch! I will take a look when I have a moment, thanks for reporting!

**Comment by jhadvig** [comment]
@jeremyfa thanks so much ! :)

**Comment by jwforres** [comment]
Same thing happens if you have a comment on a newline in between array items.

```
- something
# comment
- something else
```
